<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Menu_Principal</class>
 <widget class="QDialog" name="Menu_Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Men√∫ Principal</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(61, 139, 255);</string>
  </property>
  <widget class="QPushButton" name="btnRegAsig">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Registrar asignatura</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegAlum">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>90</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Registrar alumnos</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnIngresarNotas">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Ingresar notas</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMostrarNotas">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 127);</string>
   </property>
   <property name="text">
    <string>Mostrar notas</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
